tests/base/test_evaluation.py

```diff
@@ -258,6 +261,31 @@ def test_run_cv_evaluation():
     assert len(entity_results.test['ner_crf']["F1-score"]) == n_folds
 
 
+def test_report_output(tmpdir_factory):
+
+    path = tmpdir_factory.mktemp("evaluation").strpath
+    report_filename = path + "report.json"
+
+    intent_results = [
+        IntentEvaluationResult("", "restaurant_search",
+                               "I am hungry", 0.12345),
+        IntentEvaluationResult("greet", "greet",
+                               "hello", 0.98765)]
+
+    result = evaluate_intents(intent_results,
+                              report_filename,
+                              successes_filename=None,
+                              errors_filename=None,
+                              confmat_filename=None,
+                              intent_hist_filename=None)
+
+    report = json.loads(utils.read_file(report_filename))
+
+    assert len(report.keys()) == 4
+    assert report["greet"]
```

Comment:
can we assert something more specific here? I mean the value should be known right?